apps/condo/domains/organization/schema/ResetOrganizationService.js

```diff
@@ -0,0 +1,184 @@
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Unsupported value for dv',
+    },
+    SENDER_MISMATCH: {
+        mutation: 'resetOrganization',
+        variable: ['data', 'sender'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Unsupported value for sender',
+    },
+    ORGANIZATION_NOT_FOUND: {
+        mutation: 'resetOrganization',
+        variable: ['data', 'organizationId'],
+        code: BAD_USER_INPUT,
+        type: 'ORGANIZATION_NOT_FOUND',
+        message: 'Could not find organization by provided id',
+    },
+}
+
+const ResetOrganizationService = new GQLCustomSchema('ResetOrganizationService', {
+    types: [
+        {
+            access: true,
+            type: 'input ResetOrganizationInput { dv: Int! sender: JSON! organizationId: ID! }',
+        },
+        {
+            access: true,
+            type: 'type ResetOrganizationOutput { status: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canResetOrganization,
+            schema: 'resetOrganization(data: ResetOrganizationInput!): ResetOrganizationOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { data: { dv, sender, organizationId } } = args
+                checkDvAndSender({ dv, sender }, { ...errors.DV_VERSION_MISMATCH, mutation: 'resetOrganization' }, { ...errors.SENDER_MISMATCH, mutation: 'resetOrganization' }, context)
+
+                const DV_SENDER = { dv, sender }
+
+                if (!organizationId) throw new Error('resetOrganization(): no organizationId')
+
+                const organizationEntity = await getById('Organization', organizationId)
+                if (!organizationEntity) {
+                    throw new GQLError(errors.ORGANIZATION_NOT_FOUND, context)
+                }
+
+                const properties = await loadListByChunks({
+                    context,
+                    list: Property,
+                    chunkSize: 20,
+                    limit: 100000,
+                    where: {
+                        deletedAt: null,
+                        organization: { id: organizationId },
+                    },
+                })
+                for (let property of properties) {
+                    await Property.softDelete(context, property.id, DV_SENDER)
+                }
+
+                const employees = await loadListByChunks({
+                    context,
+                    list: OrganizationEmployee,
+                    chunkSize: 20,
+                    limit: 100000,
+                    where: {
+                        deletedAt: null,
+                        organization: { id: organizationId },
+                    },
+                })
+                for (let employee of employees) {
+                    await OrganizationEmployee.softDelete(context, employee.id, DV_SENDER)
+                }
+
```

Comment:
You also need to delete `OrganizationEmployeeRole`

Reply:
@pahaz explained the essence of this mutation as deleting an organization with all integrations disabled, i.e. deleting their connection contexts. Removal of such OrganozationEmployeeRole and specializations will be handled by a different logic. Later, when the cascading deletion of related entities appears, when deleting an employee, these entities will be cleaned up